recipes/zodb/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "ZODB" %}
+{% set version = "5.2.4" %}
+{% set bundle = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash = "ecc1c9c1ef494a70ccfc03184bccb72421d72233e9dc4742ac58f5396d04cadf" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ bundle }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
+  {{ hash_type }}: {{ hash }}
+
+build:
+  preserve_egg_dir: True
+  entry_points:
+    - fsdump=ZODB.FileStorage.fsdump:main
+    - fsoids=ZODB.scripts.fsoids:main
+    - fsrefs=ZODB.scripts.fsrefs:main
+    - fstail=ZODB.scripts.fstail:Main
+    - repozo=ZODB.scripts.repozo:main
+
+  number: {{ build }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - setuptools
+    - persistent >=4.2.0
+    - btrees >=4.2.0
+    - zconfig
+    - transaction >=2.0.3
+    - six
+    - zc.lockfile
+    - zope.interface
+    - zodbpickle >=0.6.0
+
+test:
+  imports:
+    - ZODB
+    - ZODB.FileStorage
+    - ZODB.scripts
+    - ZODB.scripts.tests
+    - ZODB.tests
+
+  commands:
+    # - fsdump --help  # no support for --help
```

Comment:
I'm not sure if it's necessary, but I guess for the cli's that re missing a help flag, you could check for the existence of them:

http://conda-forge.github.io/docs/testing.html#simple-existence-tests

but I think that's up to you.